### DISCUSSION TITLE
Need to sort the styles by style name and not by style id

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#1145] [General]
Is it possible to change the sorting of the Styles to the name given in the .mbs file?
I tried to force the sorting by prefixing with a), b) c) etc but didn't work since the sorting is done by symbol id.

Example below:
![image](https://github.com/interactive-instruments/ldproxy/assets/1905113/59353ff1-7519-4508-833d-69423ae99b89)

**Comment by cportele** [comment]
Currently there is no option to control the order in which the styles appear (beside changing the style id / filename).
